massa-models/src/address.rs

```diff
@@ -43,7 +44,13 @@ impl std::fmt::Display for Address {
 
 impl std::fmt::Debug for Address {
     fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
-        write!(f, "{}", self)
+        write!(
+            f,
+            "{} Thread {} of {}",
+            self,
+            self.get_thread(THREAD_COUNT),
```

Comment:
Actually this might be an issue when using test configs, since the same `config::THREAD_COUNT` will always be taken, but there really isn't any way to take this value as input other than saving in `Address`(which is not an option).

Reply:
That could potentially be error inducing for a dev in the future, and we have no real solution for that.

Reply:
@AurelienFT considering this do we want to have that?

Reply:
Maybe re-implementing it in the test exports could be a solution even if it's not a great one.